### ISSUE TITLE
[FEATURE]: MAKE SIDEBAR BUTTONS ACCESSIBLE 💡

### ISSUE THREAD

**Issue by 0xabdulkhalid** [group-patronage/group-patronage#141]
### Description

- The buttons on sidebar are not accessible yet to the screen readers like devices.
- The buttons are only filled with image elements, there's no proper measures taken to announce about the buttons and it's usage to the screen readers.
- Serving accessible sites to the user is a predominant way of getting more users to our site, we must want to focus on accessibility to improve our site features.

### Screenshots

- We can't take screenshots of accessibility changes rather than code changes, Accessibility changes are often invisible to the user because it's intended to guide disabled people

**Comment by 0xabdulkhalid** [comment]
Hey there @LakshaySK106 👋🏻

Please make sure to assign this issue to me as soon possible so that i can make the work done!

Thank you.
